Makefile.am

```diff
@@ -0,0 +1,304 @@
+desktopdir = $(datarootdir)/applications
+desktop_DATA = dpkgfiles/armory.desktop dpkgfiles/armoryoffline.desktop \
+			  dpkgfiles/armorytestnet.desktop
+CLEANFILES = $(desktop_DATA)
+
+PYTHON_INCLUDES=@PYTHON_INCLUDES@
+
+LINK = $(CXX)
+
+LDLIBS = -lpthread -Lmdb
+CPPFLAGS = $(ARMORY_CPPFLAGS) -IcppForSwig/cryptopp -IcppForSwig/mdb -DUSE_CRYPTOPP -D__STDC_LIMIT_MACROS
+CXXCPP = $(CPPFLAGS) $(PYTHON_INCLUDES)
+
+if TARGET_DARWIN
+LDFLAGS += -undefined dynamic_lookup -headerpad_max_install_names
+LDLIBS += -lc++
+CXXCPP += -stdlib=libc++
+endif
+
+if TARGET_LINUX
+LDLIBS += -lrt
```

Comment:
I see. Honestly, I'm a little torn since this means somebody will have to add it back if it's ever needed. I guess we'll just leave it alone for now and cross that bridge if we ever get there.